app/controllers/CreateController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+class CreateController extends ApplicationController
+{
+	public function createAction()
+	{
+			if ($_SERVER['REQUEST_METHOD'] === 'POST') {
```

Comment:
Just a suggestion. Controller class (which is parent of ApplicationController) has public method `isPost` that returns boolean check. You could implement it here instead of `$_SERVER['REQUEST_METHOD'] === 'POST'`. It's basically the same code. 

I don't really have any explanation why `if($this->isPost())` would be better than your option, so it really is just a suggestion. 🤷‍♀️